ckanext/unaids/react/components/BulkFileUploaderComponent/src/Modal.js

```diff
@@ -3,13 +3,17 @@ import ModalBody from './ModalBody';
 
 export default function Modal(props) {
     return (
-        <div className="modal fade" id={props.modalElementId}>
+        <div className="modal fade" id={props.modalElementId} data-backdrop="static" data-keyboard="false">
             <div className="modal-dialog" role="document">
                 <div className="modal-content">
                     <div className="modal-header">
-                        <button type="button" className="close" data-dismiss="modal">
-                            <span aria-hidden="true">&times;</span>
-                        </button>
+                        {!props.uploadsComplete && !props.uploadInProgress &&
```

Comment:
Just checking you have thought through what happens if there is an error in an upload?

Reply:
![image](https://user-images.githubusercontent.com/2634482/128166308-9db8696c-898f-42dd-9d9f-a97e19e2e671.png)

Yep, this behaves as expected